modules/setup/src/main/FormFactory.scala

```diff
@@ -150,5 +150,11 @@ final class FormFactory {
         "fen" -> fenField
       )(OpenConfig.from)(_.>>).verifying("invalidFen", _.validFen)
     )
+
+    lazy val declineChallenge = Form(
+      mapping(
+        "reason" -> optional(nonEmptyText)
```

Comment:
Would it be better to use standardized string codes for the reason? Not sure if it's possible to come up with a standardized list of decline reasons.